src/index.html

```diff
@@ -22,6 +22,390 @@
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
+          />
+        </div>
+
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="code">Product code: 195434</div>
+
+        <div class="stars_4">
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_gr"></div>
+          <span class="review">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="azer">Price:</span>
+          $2,199
+        </div>
+        <button
+          class="button"
+          data-qa="card-hover"
+        >
+          <span class="by">BUY</span>
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
+          />
+        </div>
+
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="code">Product code: 195434</div>
+
+        <div class="stars_4">
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_gr"></div>
+          <span class="review">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="azer">Price:</span>
+          $2,199
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
```

Comment:
The data-qa attribute 'hover' is used here, but in another similar button, 'card-hover' is used. Ensure consistency in naming conventions for data attributes.